app/controllers/bulk_uploads_controller.rb

```diff
@@ -31,11 +31,14 @@ def update
   end
 
   def create
-    redirect_to(:back) and return if params[:file].nil?
+    redirect_to(:back) and return if params[:files].nil?
 
-    result = CsvImporter.import(params[:file].path)
-    if result[:successful]
-      redirect_to BulkUpload.create(result), flash: {success: t("bulk_uploads.upload_successful")}
+    results = params[:files].map { |file| CsvImporter.import(file.path) }
```

Comment:
Basically doing the exact same thing as before, except now creating an array of the results rather than a single result.